packages/@react-aria/utils/src/getScrollParent.ts

```diff
@@ -11,15 +11,16 @@
  */
 
 export function getScrollParent(node: Element, checkForOverflow?: boolean): Element {
-  if (isScrollable(node, checkForOverflow)) {
-    node = node.parentElement;
+  let scrollableNode: Element | null = node;
```

Comment:
Is this because we can't change the API by making it `getScrollParent(node: Element | null,` instead?

Reply:
node should be defined at this point in time in the API, but node.parentElement might be null, so that's where the type comes from